SuperTicTacToe/Classes/GameLogicRework.swift

```diff
@@ -1,26 +1,20 @@
-//
-//  GameLogicRework.swift
-//  SuperTicTacToe
-//
-//  Created by Work Mode on 2/27/20.
-//  Copyright © 2020 Apps By Ennis. All rights reserved.
-//
-
 import Foundation
 
+// I think it's great that you are breaking out game logic from a view controller.
+// I would like to see some testing against these non-UI classes.
 class BasicGame {
     
     var gameboard = [Int](repeating: 0, count: 9)
```

Comment:
Having a full Int array for these could lead to bugs re accessing spaces out of bounds. Consider making a separate simple Gameboard Type?